lectures/3-MatrixProductStates/InfiniteMPS.md

```diff
@@ -1,9 +1,347 @@
+```
+
+To ensure a properly normalized state we should therefore rescale the leading eigenvalue $\lambda_0$ to one by rescaling the MPS tensor as $A \leftarrow A / \sqrt{\lambda_0}$, as well as normalize the fixed points $l$ and $r$ by requiring their trace to be equal to one:
+
+```{image} /_static/figures/imps/traceNorm.svg
+:name: trace_norm
+:align: center
+```
+
+With these properties in place, the norm of an MPS reduces to the overlap between the boundary vectors and the fixed points. Since there is no effect of the boundary vectors on the bulk properties of the MPS, we can always choose these such that MPS is properly normalized as $ \left \langle \Psi(\bar{A})\middle | \Psi(A) \right \rangle = 1$.
+
+
+### Expectation values
+
+The fixed points of the transfer matrix can for example be to compute expectation values of operators. Suppose we wish to evaluate expectation values of an extensive operator,
+
+```{math}
+O = \frac{1}{\mathbb{Z}} \sum_{n \in \mathbb{Z}} O_n.
+```
+
+If we assume that each $O_n$ acts on a single site and we are working with a properly normalized MPS, translation invariance dictates that the expectation value of $O$ is given by the contraction
+
+```{image} /_static/figures/imps/expVal.svg
+:name: exp_val
+:align: center
+```
+
+In the uniform gauge, we can use the fixed points of the transfer matrix to contract everything to the left and to the right of the operator, such that we are left with the contraction
+
+```{image} /_static/figures/imps/expVal2.svg
+:name: exp_val2
+:align: center
+```
+
+### Correlation functions
+
+Correlation functions are computed similarly. Let us look at
+
+```{math}
+c^{\alpha\beta}(m,n) = \bra{\Psi(\bar A)} (O^\beta_m)^\dagger O^\alpha_n \ket{\Psi(A)},
+```
+
+where $m$ and $n$ are abritrary locations in the chain, and, because of translation invariance, the correlation function only depends on the difference $m-n$. Again, we contract everything to the left and right of the operators by inserting the fixed points $l$ and $r$, so that
+
+```{image} /_static/figures/imps/corrFunc.svg
+:name: corr_func
+:align: center
+```
+
+From this expression, we learn that it is the transfer matrix that determines the correlations in the ground state. Indeed, if we again use the spectral decomposition of the transfer matrix, recalling that now $\lambda_0 = 0$, we can see that the correlation function reduces to
+
+```{image} /_static/figures/imps/corrFunc2.svg
+:name: corr_func2
+:align: center
+```
+
+The first part is just the product of the expectation values of $O^\alpha$ and $O^\beta$, called the disconnected part of the correlation function, and the rest is an exponentially decaying part. This expression implies that connected correlation functions of an MPS *always* decay exponentially, which is one of the reasons why MPS are not well suited for capturing critical states. The correlation length $\xi$ is determined by the second largest eigenvalue of the transfer matrix $\lambda_1$ as
```

Comment:
I would slightly reword the sentence "MPS are not well suited for capturing critical states", into something like "MPS have a harder time dealing with critical states", as we do (succesfully) use MPS for critical states.

Reply:
Good point. The line is tweaked accordingly and I also circled back to this in the note about finite entanglement scaling.